incubator/foundry-vtt/values.yaml

```diff
@@ -163,7 +163,7 @@ foundryvtt:
   # license_key:
 
   ## Admin password. Default: admin
-  adminKey: admin
+  adminPassword: admin
 
   ## A custom hostname to use in place of the host machine's public IP address when displaying the address of the game session. This allows for reverse proxies or DNS servers to modify the public address.
   hostname: "foundry.domain.com"
```

Comment:
I notice we set the hostname at multiple points in the `values.yaml`. Maybe we can reduce that by bundling some of these stanzas into a `global` mapping or something?

Reply:
Nice I'll check those this week and update.